### ISSUE TITLE
Text layer won't show

### ISSUE THREAD

**Issue by timeIoop** [photopea/photopea#6999]
I can type out something but it shows up as invisible. This is only on the google chrome version of photopea as I tried it on a different browser and it works, but i use a chromebook so chrome is easier to use it on and I'd rather not have to switch to a different browser. It won't even let me highlight the text to change the font or resize it or anything. The app version also won't show the text but I believe that's connected to chrome still so that checks out. The screen recording is for the website version.

[Screen recording 2024-07-25 17.31.37.webm](https://github.com/user-attachments/assets/13baf72e-19f9-45a8-9bc4-9d03c699289d)

**Comment by photopea** [comment]
I think you installed a browser extension in your chrome, which breaks the text feature of Photopea.

It works in other web browsers, because you did not install that browser extension into your other browsers.

Please tell us if you have any problems with Photopea, which is being used in a browser with no browser extensions modifying it.

**Comment by photopea** [comment]
I have to close it because the pesron did not respond :(

**Comment by timeIoop** [comment]
Apologies for the late response - I'm aware this is closed but hope that
this will still potentially reach someone - I don't believe that's the case
as I don't have any browser extension installed that would affect the text
feature on photopea - the only things i have are an adblocker that i
disable when using photopea, a few coupon extensions, google translate,
grammarly, and one or two other things that shouldn't have anything to do
with photopea. I've noticed it keeps glitching out on the text file now
like there's a jammed key or something, but i don't think that is the case?
I typed out 'hello' as a test run and it does this instead when I click
off. '[image: image.png] which has never happened before with photopea.
